examples/data/r54_9.mm

```diff
@@ -0,0 +1,95 @@
+************************************************************************
```

Comment:
This instance is identical to the one from RCPSP but we need to include "renewable" fields when parsing.